src/com/nutrons/nu17/RobotMap.java

```diff
@@ -36,18 +36,18 @@
 	public static final int MICRO_SWITCH = 0;
 
 	// Ports of encoders
-	public static final int DRIVE_DISTANCE_ENCODER_PORT_1 = 1;
-	public static final int DRIVE_DISTANCE_ENCODER_PORT_2 = 2;
+	public static final int DRIVE_DISTANCE_ENCODER_1 = 1;
```

Comment:
1 and 2 are not helpful Josh.  Once again, I should be able to look at the variable and find it on the Robot.  1 and 2 have no meaning on the robot.